src/components/site/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,208 @@
+                        placeholder="Origin Bus Stop..."
+                        required
+                      />
+                    }
+                  </div>
+                  <div id="destination">
+                    { this.state.isSubmitted ?
+                      <input
+                        data-testid="destination"
+                        type="text"
+                        name="destination"
+                        value={this.state.destination}
+                        className="search-fields form-control text-center"
+                        placeholder="Destination Bus Stop..."
+                        disabled
+                      />
+                      :
+                      <input
+                        data-testid="destination"
+                        type="text"
+                        name="destination"
+                        value={this.state.destination}
+                        onChange={this.handleChange}
+                        className="search-fields form-control"
+                        placeholder="Destination Bus Stop..."
+                        required
+                      />
+                    }
+                  </div>
+                </div>
+                </div>
+              <button
+                type="submit"
+                data-testid="search-btn"
+                onClick={this.handleClick}
+                style={{marginTop: "1.3em"}}
+                name="search-btn"
+                className="btn btn-info btn-block">Search</button>
+              </form>
+            </div>
+          </div>
+          <div data-testid="bus-info" id="bus-details">
+            { this.state.isSubmitted ? 
+              <div>
+                <small className="form-text text-muted">(Sample Data)</small>
+                <div>Nearby Bus</div>
+                <div className="bus-info">
+                  <div className="bus-detail-wrapper">
+                    <div><i className="fa fa-clock-o fa-lg"></i></div>
+                    <div>
+                      <div className="text-info time-remaining">10 min</div>
+                      <div className="bus-detail">away from {this.state.origin} Bus Stop</div>
+                    </div>
+                  </div>
+                  <div className="bus-detail-wrapper">
+                    <div><i className="fa fa-bus fa-lg"></i></div>
+                    <div className="bus-detail">RAC43353</div>
+                  </div>
+                  <div className="bus-detail-wrapper">
+                    <div><i className="fa fa-map-marker fa-lg"></i></div>
+                    <div className="bus-detail">Nyabugogo - {this.state.origin} - {this.state.destination}</div>
+                  </div>
+                </div>
+              </div>
+              : this.state.missingOrigin && this.state.missingDest? <h3 className="text-center text-danger">You need to provide your origin and destination</h3>
```

Comment:
instead of displaying the error message like his when a field is required, it's better to just use the attribute *required* on the input fields